pkg/noun/events.c

```diff
@@ -1038,6 +1038,13 @@ _ce_backup(void)
 
   close(nop_u.fid_i);
   close(sop_u.fid_i);
+  fprintf(stderr, "loom: image backup complete\r\n");
+}
+
+c3_o
+u3e_curr(void)
+{
+  return __(0 == _ce_patch_open());
```

Comment:
This will leak memory in the case where a patch does exist -- needs to be coupled with `_ce_patch_free(pat_u)`

Reply:
Thanks, fixed.